src/llama_stack_provider_ragas/eval_remote.py

```
@@ -228,14 +244,12 @@ async def job_status(self, benchmark_id: str, job_id: str) -> Job:
-        s3_url = "s3://public-rhods/ragas-evaluation-pipeline/results.jsonl"
-
         try:
-            df = pd.read_json(s3_url, lines=True)
-            logger.info(f"Successfully fetched results from {s3_url}")
+            df = pd.read_json(job.result_s3_location, lines=True)
+            logger.info(f"Successfully fetched results from {job.result_s3_location}")
         except Exception as e:
             raise RagasEvaluationError(
-                f"Failed to fetch results from {s3_url}: {str(e)}"
+                f"Failed to fetch results from {job.result_s3_location}: {str(e)}"
             ) from e
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)